spec/models/miq_request_spec.rb

```diff
@@ -5,20 +5,21 @@
-        :MiqProvisionRequestTemplate              => {:template                        => "VM Provision Template"},
-        :MiqProvisionConfiguredSystemRequest      => {:provision_via_foreman           => "#{ui_lookup(:ui_title => 'foreman')} Provision"},
-        :VmReconfigureRequest                     => {:vm_reconfigure                  => "VM Reconfigure"},
-        :VmCloudReconfigureRequest                => {:vm_cloud_reconfigure            => "VM Cloud Reconfigure"},
-        :VmMigrateRequest                         => {:vm_migrate                      => "VM Migrate"},
-        :VmRetireRequest                          => {:vm_retire                       => "VM Retire"},
-        :ServiceRetireRequest                     => {:service_retire                  => "Service Retire"},
-        :OrchestrationStackRetireRequest          => {:orchestration_stack_retire      => "Orchestration Stack Retire"},
-        :AutomationRequest                        => {:automation                      => "Automation"},
-        :ServiceTemplateProvisionRequest          => {:clone_to_service                => "Service Provision"},
-        :ServiceReconfigureRequest                => {:service_reconfigure             => "Service Reconfigure"},
-        :PhysicalServerProvisionRequest           => {:provision_physical_server       => "Physical Server Provision"},
-        :PhysicalServerFirmwareUpdateRequest      => {:physical_server_firmware_update => "Physical Server Firmware Update"},
+        :MiqProvisionRequest                      => {:template                         => "VM Provision", :clone_to_vm => "VM Clone", :clone_to_template => "VM Publish"},
+        :MiqProvisionRequestTemplate              => {:template                         => "VM Provision Template"},
+        :MiqProvisionConfiguredSystemRequest      => {:provision_via_foreman            => "#{ui_lookup(:ui_title => 'foreman')} Provision"},
+        :VmReconfigureRequest                     => {:vm_reconfigure                   => "VM Reconfigure"},
+        :VmCloudReconfigureRequest                => {:vm_cloud_reconfigure             => "VM Cloud Reconfigure"},
+        :VmMigrateRequest                         => {:vm_migrate                       => "VM Migrate"},
+        :VmRetireRequest                          => {:vm_retire                        => "VM Retire"},
+        :ServiceRetireRequest                     => {:service_retire                   => "Service Retire"},
+        :OrchestrationStackRetireRequest          => {:orchestration_stack_retire       => "Orchestration Stack Retire"},
+        :AutomationRequest                        => {:automation                       => "Automation"},
+        :MiqProvisionConfigurationScriptRequest   => {:provision_via_automation_manager => "Automation Manager Provision"},
```

Comment:
Will we use this same type for floe?
Is this name still good for that?
(if "yes, no", then `:provision_via_automation`?)

Reply:
Hey so my plan was to use the same request type for all AutomationManagers which would include Workflows, EmbeddedTerraform, TerraformEnterprise, and AWX/AnsibleTower since they all use some variation of ConfigurationScripts